### ISSUE TITLE
Refined Storage Fails when Exporting Lava

### ISSUE THREAD

**Issue by maraudir** [FTBTeam/FTB-Modpack-Issues#4163]
### Modpack

FTB Presents Direwolf20 1.20

### Modpack version

1.4.0

### Describe the bug

- Ender tank with lava is connected to Refined Storage using an External Storage bus. (Making it available to the system)
- Lava is exported to a Material Stoneworks machine to make obsidian.  When the machine is running items are unable to be removed from the Refined Storage system.
- Lava is exported to a Pedestal to create obsidian, this also causes the Refined Storage system to fail.

### Steps to reproduce

1. Connect a tank with lava to the Refined Storage System using an Export Interface.
2. Export Lava to either Material Stoneworks or Pedestal.
3. Access Refined Storage.
4. Unable to pull items from the system.  Will see new items added to the system not stack.

### Expected behaviour

Should be able to access the Refined Storage system freely despite exporting lava.

### Screenshots

https://imageupload.io/cEnoF8h2dlktKme
https://imageupload.io/WgvZZKWz5uTeQZW

**Comment by Skilles** [comment]
I am exporting lava to my obsidian pedestal in RF with no problem (though I do have to enter the Nether every time my save reloads to make the ftb chunk loading work again)

**Comment by maraudir** [comment]
I'm using the Mekanism pump to pump lava from the nether into the ender tank.  The lava seems to flow but will stop the RS system from working properly until I stop the lava export.

**Comment by maraudir** [comment]
Set up a fluid storage block for lava.  Disconnected the ender tank and obsidian can be made without affecting the RS system.

**Comment by Skilles** [comment]
> I'm using the Mekanism pump to pump lava from the nether into the ender tank. The lava seems to flow but will stop the RS system from working properly until I stop the lava export. Would like to note that this is on a server instance.

For my setup, I have an ender tank next to a fluid drawer with a puller upgrade. Then I connect the drawer to my drawer controller (which I made sure is in the same chunk as my controller) which has an external storage. I have noticed that refined storage is behaving weirdly with external storages and importers when chunks get unloaded and they are in a different chunk than the controller. This works fine for so if you already have a drawer setup might be a good workaround until a fix is made
